tests/scripts/thread-cert/node.py

```diff
@@ -1623,6 +1623,12 @@ def router_table(self):
 
         return router_table
 
+    def link_metrics_query_single_probe(self, dst_addr, typeid_flags):
+        cmd = 'linkmetrics query %s single %x' % (dst_addr, typeid_flags)
+        print(cmd)
```

Comment:
nit: remove `print(cmd)`?
per `cmd` is already printed when send
https://github.com/openthread/openthread/blob/e9d578da51848f566670ae757ce95d30a8c02e36/tests/scripts/thread-cert/node.py#L387-L389